docs/tutorial.md

```diff
@@ -264,21 +270,37 @@ func populateMetrics(ms *metric.MetricSet) error {
 	if err != nil {
 		return err
 	}
-	splitedLine := strings.Split(string(output), ":")
-	if len(splitedLine) != 2 {
+	splittedLine := strings.Split(string(output), ":")
```

Comment:
This change should be also reflected in the tutorial code: https://github.com/newrelic/infra-integrations-sdk/blob/master/docs/tutorial-code/redis.go